legal-api/migrations/versions/94586877d4bc.py

```diff
@@ -0,0 +1,29 @@
+"""empty message
+
+Revision ID: 94586877d4bc
+Revises: 9147d4bc9939
+Create Date: 2022-02-16 06:07:35
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = '94586877d4bc'
+down_revision = '9147d4bc9939'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
```

Comment:
Please check with Thor whether we add dml as part of the migration or it has to be a sql script on its own

Reply:
Seems small enough that we could leave it in.

We may want to cut a major version and redo our migration scripts to be just 1 new starting file. Probably something for us to huddle on.